config/initializers/github_client.rb

```diff
@@ -2,12 +2,22 @@
 
 module GitHubClassroom
   def self.github_client(options = {})
-    client_options = {
-      client_id:     Rails.application.secrets.github_client_id,
-      client_secret: Rails.application.secrets.github_client_secret,
-      auto_paginate: true
-    }.merge!(options)
+    client_options = options.tap do |opts|
+      opts[:user_agent] = "GitHub Classroom"
```

Comment:
Why do we need to set this now?

Reply:
See [this comment from @tarebyte](https://github.com/education/classroom/pull/857#discussion_r110309411) from my previous PR on this issue.